src/clients/python/examples/simple_grpc_infer_client.py

```diff
@@ -78,6 +78,15 @@
         required=False,
         default=None,
         help='File holding PEM-encoded certicate chain. Default is None.')
+    parser.add_argument(
+        '-C',
+        '--compression_algorithm',
```

Comment:
Since this is just for GRPC maybe call it `--grpc-compression-algorithm` instead?

Reply:
The client is itself grpc hence I omitted grpc here... But I can make this change for uniformity with perf_analyzer ( and perf_analyzer in the future)

Reply:
Yes, I agree that having uniform flags will help 😄